### ISSUE TITLE
not able to replicate the results

### ISSUE THREAD

**Issue by dipendra009** [d-li14/mobilenetv3.pytorch#24]
Hi,
I am running your implementation with the hyper parameters you provided using the examples/imagenet/main.py provided by PyTorch. Seems like the accuracy metrics are stuck around after epoch 20. Could you please share your training script with hyper parameter settings?

**Comment by d-li14** [comment]
Please refer to https://github.com/d-li14/mobilenetv2.pytorch/blob/master/imagenet.py for the training script. Also, the training curve with cosine LR decay should look like this https://github.com/d-li14/ghostnet.pytorch/raw/master/figs/ablation.png, in which the accuracy grows slowly in the middle, but is not stuck.
